scripts/py_matter_yamltests/matter_yamltests/runner.py

```diff
@@ -175,6 +180,9 @@ async def _run(self, parser: TestParser, config: TestRunnerConfig):
 
             test_duration = 0
             for idx, request in enumerate(parser.tests):
+                if request.is_event and request.is_last_event_number:
```

Comment:
`is_last_event_number` is an odd name. I expect "is" to mean a property of something, where as this seems to control the request itself. maybe "use_last_event_number" or "new_events_only" or something like that.

Reply:
I think new_events_only is more appropriate, because it may actually have more than one.